website/docs/installation/offline.mdx

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 0
+---
+
+# Offline
+
+This guide explains how to self-host Tabby offline.
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+```bash title="online.sh"
+# On an internet connected machine
+docker run --rm --name tabby -it -p 8080:8080 -v $HOME/models:/data tabbyml/tabby download --model TabbyML/StarCoder-1B
+tar zcvf models.tgz $HOME/models
+docker save -o tabbyml.tar tabbyml/tabby
```

Comment:
Probably add a new faq inside of https://github.com/TabbyML/tabby/edit/main/website/docs/faq.mdx and explain that the operator needs to transfer mounted directory? It'll be slightly easier to understand - as other part of this guide is somewhat covered by the docker installation guide.

Reply:
How does this amendment look?